src/explore-education-statistics-admin/src/pages/release/pre-release/PreReleasePageContainer.tsx

```diff
@@ -104,6 +104,18 @@ const PreReleasePageContainer = ({
     }
 
     if (access === 'Before') {
+      const zonedStartDate = utcToZonedTime(start, 'Europe/Berlin');
+      const dateOfPraStart = format(
+        utcToZonedTime(zonedStartDate, 'Europe/London'),
```

Comment:
Do we need to do `utcToZonedTime` again here?

Reply:
We absolutely do not, no :) Will remove!